test/testUnregisterMachine.js

```diff
@@ -67,26 +92,29 @@ describe('mercurius unregisterMachine', function() {
       token: token,
       machineId: 'machine_2',
     })
-    .expect(200, done);
+    .expect(404, done);
```

Comment:
I think sendMachines function shouldn't send 404 at all.
It should be done before in route and send machines should just do that...